src/status_helper.js

```diff
@@ -1,23 +1,15 @@
 'use strict'
 
-function pendingDescription (release, runningRelease, runningSlug) {
-  if (runningRelease && runningRelease.id === release.id && hasReleasePhase(runningSlug)) {
-    return 'release command executing'
-  } else {
-    return 'pending'
-  }
-}
-
 function hasReleasePhase (slug) {
```

Comment:
this is where the linter is failling, did you intend to use this?

Reply:
Indeed, it's not used anymore and I missed that. I've removed it.